tests/virtual_machines_test.go

```diff
@@ -33,6 +36,24 @@ var _ = Describe("Virtual Machines", func() {
 	})
 
 	Context("Check the client", func() {
+		BeforeEach(func() {
+			// Verify that there are no VMs in the cluster
```

Comment:
is there a scenario where this check will find a live vm in the cluster? if the answer is no then I think it should be dropped. 
I think the [check](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/ee7e920feb78b555e329d8ac47644bc8b0871e20/tests/virtual_machines_test.go#L73) in the AfterEach is already checking this.

Reply:
In case AfterEach delete VM part which inside an Eventually times out, a VM might be left in the cluster, then this double check will raise an error about it.